test/unit/util.test.js

```diff
@@ -68,7 +68,7 @@ describe('util', function () {
                 },
                 options = {
                     noDefaults: false,
-                    retainIds: false
```

Comment:
@codenirvana What does this change do?

Reply:
Fixed wrongly asserting test, the test title says `"retainIds is true"`.